lib/rules/no-unknown-property.js

```diff
@@ -234,7 +234,7 @@ const DOM_PROPERTY_NAMES_ONE_WORD = [
   // OpenGraph meta tag attributes
   'property',
   // React specific attributes
-  'ref', 'key', 'children',
+  'ref', 'key', 'children', 'precedence',
```

Comment:
i think maybe this needs to be removed here
```suggestion
  'ref', 'key', 'children',
```

Reply:
you were faster than me 😅